src/main/java/org/mifos/connector/mpesa/camel/config/CustomHeaderFilterStrategy.java

```diff
@@ -0,0 +1,26 @@
+package org.mifos.connector.mpesa.camel.config;
+
+import org.apache.camel.support.DefaultHeaderFilterStrategy;
+
+public class CustomHeaderFilterStrategy extends DefaultHeaderFilterStrategy {
+
+    public CustomHeaderFilterStrategy() {
+        initialize();
+    }
+
+    protected void initialize() {
+        getOutFilter().add("content-length");
```

Comment:
Do we really need all these filters? please keep only the necessary ones

Reply:
This was by default from mojaloop connector.